go/chat/supersedes.go

```diff
@@ -70,6 +70,12 @@ func (t *basicSupersedesTransform) transformEdit(msg chat1.MessageUnboxed, super
 	mvalid.AtMentionUsernames = superMsg.Valid().AtMentionUsernames
 	mvalid.ChannelMention = superMsg.Valid().ChannelMention
 	mvalid.ChannelNameMentions = superMsg.Valid().ChannelNameMentions
+	if superMsg.Valid().SenderDeviceName != "" {
```

Comment:
What is the point of this blank check?

Reply:
Because there will be older clients that send superseding messages without a device name?

Reply:
Don't think so, that is set later on in the pipeline. See my first comment, setting it there was just old code that wasn't needed.

Reply:
ok will remove